launcher/cli/logging.go

```diff
@@ -100,6 +104,58 @@ func setupLogger() {
 
 	// Hijack standard Golang `log` and redirect it to our common logger
 	zap.RedirectStdLogAt(commonLogger, zap.DebugLevel)
+
+	if listenAddr != "" {
+		go func() {
+			userLog.Debug("starting atomic level switcher on" + listenAddr)
```

Comment:
`Debug` should use standard zap formatting
```suggestion
			userLog.Debug("starting atomic level switcher http server", zap.String("listen_addr", listenAddr))
```